docs/docs/module_guides/observability/index.md

```diff
@@ -424,6 +424,54 @@ import llama_index.core
 llama_index.core.set_global_handler("simple")
 ```
 
+### Simple (LLM Inputs/Outputs)
+
+This simple observability tool prints every LLM input/output pair to the terminal. Most useful for when you need to quickly enable debug logging on your LLM application.
+
+#### Usage Pattern
+
+```python
+import llama_index.core
+
+llama_index.core.set_global_handler("simple")
+```
+
+### MLflow
+[MLflow](https://mlflow.org/docs/latest/index.html) is an open-source platform, purpose-built to assist machine learning practitioners and teams in handling the complexities of the machine learning process. MLflow focuses on the full lifecycle for machine learning projects, ensuring that each phase is manageable, traceable, and reproducible.
+
+##### Install
+```shell
+pip install mlflow>=2.15 llama-index>=0.10.44
+```
+
+#### Usage Pattern
+
+```python
+import mlflow
+
+mlflow.llama_index.autolog()  # Enable mlflow tracing
+
+with mlflow.start_run() as run:
+    mlflow.llama_index.log_model(
+        index,
+        artifact_path="llama_index",
+        engine_type="query",  # Logged engine type for inference
+        input_example="hi",
+        registered_model_name="my_llama_index_vector_store",
+    )
+    model_uri = f"runs:/{run.info.run_id}/llama_index"
+
+predictions = mlflow.pyfunc.load_model(model_uri).predict("hi")
```

Comment:
Apparently this page purely focus on observability. Can we replace this with normal llama-index querying without logging? I think there is a better place to put logging integration.

Reply:
Yeah good point. Will definitely defer to maintainers (but will wait for comments before redoing the gif)

Reply:
I think as discussed on slack, there isn't really a better place to put this for now